vignettes/plot_map.Rmd

```diff
@@ -99,11 +99,12 @@ autoplot(df, p = p, colour = 'red', size = 'population') +
 ```{r, message = FALSE, eval = FALSE}
 library(ggmap)
 bbox <- c(130.0, 30.0, 145.0, 45.0)
-map <- get_openstreetmap(bbox = bbox, scale = 47500000)
-p <- ggmap(map)
-autoplot(df, p = p, colour = 'red', size = 'population') + 
-  scale_size_area() +
-  theme(legend.justification = c(1, 0), legend.position = c(1, 0))
+## Error: 'get_openstreetmap' is defunct.
```

Comment:
Could you un-comment these lines since they are irrelevant to this PR?

Reply:
Actually this code block has `eval=FALSE` so any errors can be ignored for now.

Reply:
Done.

Reply:
Un-commented and pushed.